### PR TITLE
Added sample for MI

### PR DESCRIPTION
**Pull request by chandan-anjani**

### CODE REVIEW

File: ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Release/ManualInterventionSample.cs [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+            // Get a manual intervention
+            ManualIntervention manualIntervention = releaseClient.GetManualInterventionAsync(project: projectName, releaseId: this._newlyCreatedRelease.Id, manualInterventionId: this._manualInterventions.FirstOrDefault().Id).Result;
+            Console.WriteLine("{0} {1}", manualIntervention.Id.ToString().PadLeft(6), manualIntervention.Name);
+
+            return manualIntervention;
+        }
+
+        [ClientSampleMethod]
+        public ManualIntervention UpdateManualIntervention()
```
Comment (bansalaseem) [inline ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Release/ManualInterventionSample.cs]: Have a sample for resume and sample for reject?

File: ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Release/ManualInterventionSample.cs [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+            // Update a manual intervention
+            ManualIntervention manualIntervention = releaseClient.UpdateManualInterventionAsync(manualInterventionUpdateMetadata: manualInterventionUpdateMetadata, project: projectName, releaseId: this._newlyCreatedRelease.Id, manualInterventionId: this._manualInterventions.FirstOrDefault().Id).Result;
+            Console.WriteLine("{0} {1}", manualIntervention.Id.ToString().PadLeft(6), manualIntervention.Name);
+
+            return manualIntervention;
+        }
+
+        [ClientSampleMethod]
+        public void DeleteReleaseDefinitionWithManualIntervention()
```
Comment (bansalaseem) [inline ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Release/ManualInterventionSample.cs]: Remove this

File: ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Release/ManualInterventionSample.cs (hunk 1 above)
Comment (chandan-anjani) [reply ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Release/ManualInterventionSample.cs]: done

File: ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Release/ManualInterventionSample.cs (hunk 2 above)
Comment (chandan-anjani) [reply ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Release/ManualInterventionSample.cs]: this is needed for cleanup

**Pull request closed without merging**